### DISCUSSION TITLE
App is showing device names for partitions, should show Volemt label or mountpoint

### DISCUSSION THREAD

**Discussion by Mart-Bogdan** [spacedriveapp/spacedrive#1535] [Feature Request]
![image](https://github.com/spacedriveapp/spacedrive/assets/775038/2ad60faf-44fe-4bd1-85b8-bf99aad08248)

Currently it shows on a left partition ids, but would be nicer to display mountpoints for ones that have mountpoints.
And also take partition label, if it is available.

**Comment by utkubakir** [comment]
Created an issue for this internally (eng-1394) and we will discuss and take action :) Thanks @Mart-Bogdan
